admin_interface/static/admin_interface/css/tabbed-changeform.css

```diff
@@ -3,6 +3,7 @@
     flex-direction: row;
     flex-wrap: nowrap;
     overflow-x: auto;
+    padding-bottom: 10px;
```

Comment:
It's also possible to reduce the scrollbar size on browsers that support the `scrollbar-width` CSS option: https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width

Reply:
Updated to use scrollbar-width for firefox ; `-webkit-scrollbar-thumb` & `-webkit-scrollbar` for the rest. (tested on chrome)